data.tf

```diff
@@ -79,3 +79,13 @@ data "aws_iam_instance_profile" "custom_worker_group_launch_template_iam_instanc
 }
 
 data "aws_partition" "current" {}
+
+data "aws_eks_cluster" "cluster" {
+  count = var.create_eks ? 1 : 0
+  name  = concat(aws_eks_cluster.this[*].id, [""])[0]
```

Comment:
This is the trick to stop data sources from appearing in the plan output when modifying a parent resource.

Reply:
terraform magic again. Can you please add it as a comment, so we won't forget it ?

Reply:
👍